src/components/About.story.js

```diff
@@ -0,0 +1,78 @@
+import { storiesOf } from '@storybook/react';
+import overviewWrapper from '../storybook-addons/overviewWrapper';
+
+const stories = storiesOf('About', module);
+
+stories.add(
+    'Welcome',
+    overviewWrapper(`
+
+UI React is the Mudano component library.
+
+If you have questions ask the development team via [GitHub Issues](https://github.com/Mudano/ui-react/issues)
+
+### Installation
+
+    yarn add @mudano/ui-react
+
+_NB: installation from NPM is not yet available_
+
+### Requirements
+
+UI React components are only compatible with a React application and require the following:
+
+    node >=8.5.0
+    react >=0.14.0 <= 15
+    react-dom >=0.14.0 <= 15
+
+You'll also need the prop-types package:
+
+    yarn add prop-types
+
+If you are using the native ES6 modules, you'll need to compile your application, we recommend webpack.
+
+    yarn add babel-core babel-loader css-loader node-sass sass-loader style-loader webpack --dev
+
+### Usage
+
+You can consume UI React in one of two ways, as a whole library or as individual components.
+
+    import { YourComponent } from '@mudano/ui-react'; // UMD Modules
+
+    import { YourComponent } from 'node_modules/@mudano/ui-react/es/index'; // Native ES6 Modules
+
+    import YourComponent from 'node_modules/@mudano/ui-react/es/components/YourComponent/YourComponent';
+
```

Comment:
Were you still going to make changes to this section?

Reply:
I think we should... I'll do that now